store/sqlstore/upgrade.go

```diff
@@ -865,6 +867,14 @@ func upgradeDatabaseToVersion5281(sqlStore SqlStore) {
 	}
 }
 
+func upgradeDatabaseToVersion530(sqlStore SqlStore) {
+	if shouldPerformUpgrade(sqlStore, VERSION_5_29_0, VERSION_5_30_0) {
```

Comment:
You shouldn't add the version here, the and the `shouldPerformUpgrade` check is comment until the release is create, so... we add all the migrations for the version in the same block.